flow/src/main/java/com/sequenceiq/flow/reactor/LoggingErrorHandler.java

```diff
@@ -1,34 +0,0 @@
-package com.sequenceiq.flow.reactor;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.stereotype.Component;
-
-import reactor.fn.Consumer;
-
-/**
- * Generic error handler for the Cloudbreak application.
- * Errors (Exceptions) need to be delegated to the error translator  / interpreter subsystem.
- * <br>
- * The main functionality of the subsystem is to provide meaningful error messages to the calling systems.
- */
-@Component
-public class LoggingErrorHandler implements Consumer<Throwable> {
```

Comment:
is this replaced? or it's not necessary anymore? I don't see any info about it in the commit msg

Reply:
This is not needed anymore. I marked it in the commit message as well.